src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -195,8 +195,6 @@ async Task ApplyHotReloadUpdateInternalAsync()
 
     public Task ActivateSessionAsync(IVsLaunchedProcess? launchedProcess, VsDebugTargetProcessInfo vsDebugTargetProcessInfo)
     {
-        Assumes.True(_pendingSessionState is not null, "No pending hot reload session to activate.");
-
```

Comment:
Sad to see validation removed.

Why is this PR needed? I thought we fixed this in https://github.com/dotnet/project-system/pull/9835

Reply:
I see, this was disabled via a different means. Will investigate.